hkmc2/shared/src/main/scala/hkmc2/semantics/Resolver.scala

```diff
@@ -95,7 +95,7 @@ object Resolver:
     def showEnv: Str =
       iEnv.values
         .flatMap(_.map((typ, instance) => s"${typ.show}"))
-        .mkString("(", ", ", ")")
+        .mkString("[", ", ", "]")
```

Comment:
Why the insistence on brackets? It looks like an array now, but is not an array. Wouldn't it look better as:
```suggestion
        .mkString(", ")
```